.travis.yml

```diff
@@ -23,10 +29,14 @@ before_script:
   - phpenv config-rm xdebug.ini
 
 install:
+  - if [[ $test_jwt != 'yes' ]]; then
+    composer remove --no-update --dev lcobucci/jwt;
+  fi
```

Comment:
requires php ^7.4 || ^8.0

Reply:
resolved: added support for lcobucci/jwt v3 in order to allow using it with php >= 5.5